site/en/r2/tutorials/load_data/csv.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>You have mentioned these arguments.</p><p><br></p><p>Also, remove the arguments block, the code demonstrates it and they are pretty easy to understand based on their name.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done.</p>